src/main/java/org/broadinstitute/hellbender/utils/Utils.java

```diff
@@ -1137,4 +1138,50 @@ public static void forceJVMLocaleToUSEnglish() {
         final List<T> sorted = values.stream().sorted().collect(Collectors.toList());
         return sorted.get(sorted.size() / 2);
     }
+
+    /**
+     * Splits a String using indexOf instead of regex to speed things up.
+     *
+     * @param str the string to split.
+     * @param delimiter the delimiter used to split the string.
+     * @return A {@link String} Array of tokens.
+     */
+    public static String[] split(final String str, final String delimiter) {
+        // This is 10 because the ArrayList default capacity is 10 (but private).
+        return split(str, delimiter, 10);
+    }
+
+    /**
+     * Splits a given {@link String} using {@link String#indexOf} instead of regex to speed things up.
+     * @param str The {@link String} to split.
+     * @param delimiter The delimiter used to split the {@link String}.
+     * @param expectedNumTokens The number of tokens expected (used to initialize the capacity of the {@link ArrayList}).
+     * @return A {@link String} Array of tokens.
+     */
+    public static String[] split(final String str, final String delimiter, final int expectedNumTokens) {
+        final ArrayList<String> result =  new ArrayList<>(expectedNumTokens);
+
+        if ( str.length() == 0) {
+            result.add("");
+        }
+        else if ( delimiter.length() == 0) {
+            for ( int i = 0; i < str.length(); ++i ) {
+                result.add( str.substring(i, i+1) );
+            }
+        }
+        else if ( delimiter.length() == 1) {
+            return ParsingUtils.split(str, delimiter.charAt(0)).toArray(new String[]{});
```

Comment:
Return a List if possible, to avoid this wasteful conversion. If this would cause problems for client code, then have two versions: an array version and a List version.